chalicelib/checks/wrangler_checks.py

```diff
@@ -1368,42 +1368,50 @@ def check_for_ontology_updates(connection, **kwargs):
+        owl = None
         if o['ontology_prefix'] == 'UBERON':
-            uberon = requests.get('http://svn.code.sf.net/p/obo/svn/uberon/releases/')
-            ub_release = uberon._content.decode('utf-8').split('</li>\n  <li>')[-1]
-            versions['UBERON']['latest'] = ub_release[ub_release.index('>') + 1: ub_release.index('</a>')].rstrip('/')
-        # instead of repos etc, check download url for ontology header to get version
+            # UBERON needs different URL for version info
+            owl = requests.get('http://purl.obolibrary.org/obo/uberon.owl', headers={"Range": "bytes=0-2000"})
         elif o.get('download_url'):
+            # instead of repos etc, check download url for ontology header to get version
             owl = requests.get(o['download_url'], headers={"Range": "bytes=0-2000"})
-            if owl.status_code == 404:
-                versions[o['ontology_prefix']]['latest'] = 'WARN: 404 at download_url'
-                check.summary = '404 at download_url'
-                check.description = 'One or more ontologies has a download_url with a 404 error.'
-                check.description += ' Please update ontology item or try again later.'
-                check.status = 'WARN'
+
+        if not owl:
+            # there is an issue with the request beyond 404
+            versions[o['ontology_prefix']]['latest'] = 'WARN: no owl returned at request'
+            check.summary = 'Problem with ontology request - nothing returned'
+            check.description = 'One or more ontologies has nothing returned from attempted request.'
+            check.description += ' Please update ontology item or try again later.'
+            check.status = 'WARN'
+            continue
+        elif owl.status_code == 404:
+            versions[o['ontology_prefix']]['latest'] = 'WARN: 404 at download_url'
+            check.summary = 'Problem 404 at download_url'
+            check.description = 'One or more ontologies has a download_url with a 404 error.'
+            check.description += ' Please update ontology item or try again later.'
+            check.status = 'WARN'
+            continue
+        if 'versionIRI' in owl.text:
+            idx = owl.text.index('versionIRI')
+            vline = owl.text[idx:idx+150]
+            if 'releases'in vline:
+                vline = vline.split('/')
+                v = vline[vline.index('releases')+1]
+                versions[o['ontology_prefix']]['latest'] = v
                 continue
-            if 'versionIRI' in owl.text:
-                idx = owl.text.index('versionIRI')
-                vline = owl.text[idx:idx+150]
-                if 'releases'in vline:
-                    vline = vline.split('/')
-                    v = vline[vline.index('releases')+1]
+            else:
+                # looks for date string in versionIRI line
+                match = re.search('(2[1-5])?([0-9]{2})-[0-9]{2}-(2[1-5])?[0-9]{2}', vline)
```

Comment:
This regex needs to be reverted to the previous one, I think. I can't successfully pick up dates such as '2021-06-05' with this.